circle.yml

```diff
@@ -9,11 +9,12 @@ dependencies:
   pre:
     - pip install --upgrade pip
     - pip install Cython --install-option="--no-cython-compile"
-    - pip install numpy==1.11.1 
+    - pip install numpy==1.11.1
     - pip install scipy==0.17.1
     - pip install pytest==3.0.4
+    - pip install flake8==3.3.0
 
 test:
   override:
     - py.test --verbose
-
+    - flake8 ./ --max-line-length=79
```

Comment:
Looks like this is testing the entire codebase via flake8. It may make slightly more sense to test only the changes for flake8. This way you're not responsible for unrelated code to your PR and we can violate flake8 as needed without breaking the builds.

I know this is how pandas does it, if you want to find the commands from them.